strigo/strigo-sdk
# Add an assessment player debug mode UI

## Context
Add an assessment player debug mode UI

## Tests
- [x] Change the assessment debug toggle in Lounge to "on"
* See a deteced location element highlitened 

- [x] Change the assessment debug toggle in Lounge to "off"
* See a deteced location element  NOT highlitened
